debian/rules

```diff
@@ -39,14 +41,21 @@ override_dh_auto_install:
 	mkdir -p "$(APACHE_DIR)/apps/dev"
 	mkdir -p "$(CONFIG_DIR)/clusters.d"
 	mkdir -p "$(CONFIG_DIR)/ondemand.d"
-	mkdir -p "$(DESTDIR)/etc/sudoers.d"
 
 	mv $(DESTDIR)/opt/ood/apps/* $(APACHE_DIR)/apps/sys
 	rm -rf $(DESTDIR)/opt/ood/apps
 
+	# ood config files
 	rake package:version > $(DESTDIR)/opt/ood/VERSION
-
 	install -m 644 nginx_stage/share/nginx_stage_example.yml $(CONFIG_DIR)/nginx_stage.yml
 	install -m 640 ood-portal-generator/share/ood_portal_example.yml $(CONFIG_DIR)/ood_portal.yml
-	install -m 440 lib/tasks/files/sudo.debian $(DESTDIR)/etc/sudoers.d/ood
+	install -D -m 644 hooks/hook.env.example $(CONFIG_DIR)/ood/config/hook.env
+
+	# extra infrastructure files
+	install -D -m 440 lib/tasks/files/sudo.debian $(DESTDIR)/etc/sudoers.d/ood
 	install -m 440 lib/tasks/files/profile.debian $(DESTDIR)/etc/ood/profile
+	install -D -m 644 lib/tasks/files/crontab.debian $(DESTDIR)/etc/cron.d/ood
+	install -D -m 644 lib/tasks/files/logrotate.debian $(DESTDIR)/etc/logrotate.d/ood
+	install -D -m 644 lib/tasks/files/systemd.ood.conf.debian $(APACHE_SYSTEMD_DIR)/ood.conf
```

Comment:
The location for `APACHE_SYSTEMD_DIR` won't exist by default, so I think you may have to do something like this:

```
install -d $(APACHE_SYSTEMD_DIR)

OR

mkdir -p $(APACHE_SYSTEMD_DIR)
```

I'm not real familiar with Debian packaging but with RPMs the parent directory has to exist during packaging in order to add a file to that directory and so very common to have to do a bunch of `install -d` or `mkdir -p`.

This issue of parent directory might be necessary for the `/etc/logrotate.d` and `/etc/cron.d/` and `/etc/sudoers.d` unless the packages that add those directories are present during build.

Reply:
I thought that's what `install -D` does?

Reply:
You're correct, that will make `mkdir` or `install -d` not necessary.